manifests/service.pp

```diff
@@ -4,11 +4,16 @@
 class candlepin::service {
   assert_private()
 
+  package { ['java-11-openjdk']:
```

Comment:
Installing a package in the service class is not a good design. Usually you keep that to install.pp. That probably already gives you the dependency chaining.

I would also make it a parameter. Hardcoding it when there's often a version number is usually a problem at some point.

Another thing is that you may want to use `ensure_packages` since that deals with duplicate resources, in case something else also ensures the same package.